### ISSUE TITLE
Echo doesn't return value ending in a trailing newline

### ISSUE THREAD

**Issue by gvn** [shelljs/shelljs#476]
### Node version:

v6.2.0
### ShellJS version (the most recent version/Github branch you see the bug on):

0.7.0
### Operating system:

OSX 10.11.5
### Description of the bug:

Using `toEnd` doesn't prepend a newline when chained to `echo`.
### Example ShellJS command to reproduce the error:

``` javascript
shell.echo(`A`).toEnd(`test.txt`);
shell.echo(`B`).toEnd(`test.txt`);
```

test.txt becomes: `AB`

``` bash
echo A >> test.txt
echo B >> test.txt
```

test.txt becomes:

```
A
B
```

**Comment by nfischer** [comment]
This has to do with the behavior of `echo()`, not `toEnd`. This is still a valid point, but to fix it would involve breaking the behavior of `echo()`, so it would have to wait for v0.8.

@gvn The fix is pretty trivial, it just involves appending a newline to the value returned by `echo()`, if you'd like to fix it yourself :smile:

**Comment by gvn** [comment]
Ok cool. Yeah, would be good to fix, but I can add a newline manually for now. Thanks!

**Comment by nfischer** [comment]
@freitagbr do you want to take a look at this?

**Comment by freitagbr** [comment]
Sure, I'll take a look.

**Comment by nfischer** [comment]
This is now fixed on trunk in #557
